From 56be9d339071c71d456394445e3be79f838aab8b Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Tue, 18 Oct 2016 07:59:34 +1300
Subject: [PATCH] TOOLS-1509 Update contribution guidelines to reflect use of
 cr.joyent.us Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved by:
 Joshua M. Clulow <jmc@joyent.com>

---
 README.md | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/README.md b/README.md
index 9f8c790..b39e960 100644
--- a/README.md
+++ b/README.md
@@ -10,10 +10,10 @@
 
 # keyapi
 
-This repository is part of the Joyent SmartDataCenter project (SDC).  For 
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
-
+This repository is part of the Joyent Triton project. See the [contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
 
 KeyAPI is a simple library to create and unpackage opaque encrypted tokens
 for services that require user authentication
-- 
2.21.0

